app/helpers/analytics_helper.rb

```diff
@@ -0,0 +1,5 @@
+module AnalyticsHelper
```

Comment:
This helper seems pointless just now, but will contain some more methods once the cookies consent is implemented.